app/static/dayview.css

```diff
@@ -0,0 +1,72 @@
+
+:root {
+    --theme-blue:#30465D;
+    --theme-yellow:#FFDE4D;
+    --theme-red:#EF5454;
+    --theme-grey:#E7E7E7;
+    --theme-ligth-grey:#F7F7F7;
+}
+
+html {
+    font-family: 'Assistant', sans-serif;
+    text-align: center;
+}
+
+#phoneframe {
+    width: 260px;
```

Comment:
We should use em or %. If you want to go responsive (which is a great choice), you might want to use `@media` queries. Please Google about how to use DevTools to see how the site looks on mobile displays.

Reply:
זה שריד מההתחלה של העבודה לבדיקה של משהו, כרגע אין לו שימוש. אמחוק אותו לחלוטין